python/ray/train/BUILD

```diff
@@ -136,7 +136,7 @@ py_test(
     name = "test_batch_predictor",
     size = "small",
     srcs = ["tests/test_batch_predictor.py"],
-    tags = ["team:ml", "exclusive", "ray_air"],
+    tags = ["team:ml", "exclusive", "ray_air", "gpu"],
```

Comment:
I don't think these tests are actually getting run in CI right now.

Can we add this line to the `ML GPU Tests` in `pipeline.gpu.large.yaml`?

`- bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only --test_tag_filters=ray_air, gpu python/ray/train/...`